### PR TITLE
update README and triggerREADME

### PR DESCRIPTION
**Pull request by cachai2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,89 @@
...
-This repository contains extension code for SQL Server input and output bindings. Trigger bindings are coming soon.
+    ```sql
+    ALTER TABLE [dbo].[Employees] ADD CONSTRAINT PKey PRIMARY KEY CLUSTERED (EmployeeId);
+    ```
 
-# Input and Output Binding
-## Quick Start
+### Set Up Local .NET Function App ###
+
+**NOTE: THE MYGET PACKAGE DOES NOT CURRENTLY CONTAIN TRIGGER BINDING FUNCTIONALITY**
```
Comment (cgillum) [inline README.md:84]: This is no longer accurate, right?

File: README.md [hunk 2]
```diff
@@ -28,25 +105,124 @@ This repository contains extension code for SQL Server input and output bindings
     dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --version 1.0.0-preview1
```
Comment (cgillum) [inline README.md:105]: Should we point people to 1.0.0-preview2?

File: README.md (hunk 1 above)
Comment (cachai2) [reply README.md:84]: I had to modify .csproj in order to get the SQL trigger working. I tried it out again this morning and just using the myget package resulted in some namespaces not being found for the trigger. I've attached a screenshot below. Is the myget supposed to include full functionality for the trigger now? I might have missed something or taken a weird step in setup, so if someone can verify, that would be great. 

Capture

File: README.md (hunk 2 above)
Comment (cachai2) [reply README.md:105]: What is the difference between preview1 and preview2? @anthonychu probably has a better idea of this.

Comment (cgillum) [reply README.md:105]: preview2 is the package I uploaded that had the trigger binding.

Comment (cachai2) [reply README.md:105]: Oh ok. That makes sense. I missed that I wasn't using preview2. I was still using preview1 as I was following the README and thought it was updated. I just tried it, and trigger works. That is definitely a lot simpler. Thanks for pointing this out Chris. I'm going to close the PR for now and update the docs to reflect using preview2.

File: README.md (hunk 1 above)
Comment (cachai2) [reply README.md:84]: Fixed with preview2

**Comment by cachai2** [comment]
Closing PR for now. Updating docs to reflect using preview2.

**Pull request closed without merging**